src/app/shared/form-error-messages.component.ts

```diff
@@ -1,55 +1,50 @@
+/**
+ * Centralized component for all form error messages
+ * MUST BE WRAPPED IN A mat-error ELEMENT
+ * Takes a form control as input and outputs a span element with the error message
+ */
+
 import { Component, Input } from '@angular/core';
 import { AbstractControl, AbstractControlDirective } from '@angular/forms';
 
 @Component({
   selector: 'planet-form-error-messages',
   template: `
-    <ul class="text-danger" *ngIf="shouldShowErrors()">
-      <li *ngFor="let error of listOfErrors()">{{error}}</li>
-    </ul>
-  `,
-  styles: [
-    `ul {
-      list-style-type: none;
-      font-size: .8rem;
-      padding-left: .2em;
-    }`
-  ]
+    <span *ngIf="shouldShowError()" i18n>{updateError(), select,
+      required {This field is required}
+      min {The number cannot be below {{number}}}
```

Comment:
Here number might not be translated. And some language use different number scripts such as Hindi and Nepali use १ as 1